src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -22,7 +22,9 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
-
+{% if isColumnLocationDisplayed is not defined %}
+  {% set isColumnLocationDisplayed=(product.location is empty) %}
```

Comment:
```suggestion
  {% set isColumnLocationDisplayed=(product.location is not empty) %}
```

Reply:
But again it's cleaner to do this in the controller (if it's related to the add product feature html return)

Reply:
Fixed ;)